packages/destination-actions/src/destinations/engage/utils/MessageSendPerformer.ts

```diff
@@ -106,14 +119,16 @@ export abstract class MessageSendPerformer<
     return this.constructor as ThisStaticClass
   }
 
-  /**
-   * check if extId is (un)subscribed (returns true|false) or if subscription status is invalid (returns undefined)
-   */
+  // /**
+  //  * check if extId is (un)subscribed (returns true|false) or if subscription status is invalid (returns undefined)
+  //  */
```

Comment:
I guess you're unintentionally commented it
```suggestion
  /**
   * check if extId is (un)subscribed (returns true|false) or if subscription status is invalid (returns undefined)
   */
```